skrub/_expressions/_expressions.py

```diff
@@ -420,7 +420,8 @@ def __get__(self, instance, owner=None):
         return f"""Skrub expression.\nDocstring of the preview:\n{doc}"""
 
 
-class Expr:
+class Expr(_CloudPickle):
+    _cloudpickle_attributes = ["_skrub_impl"]
```

Comment:
Is `_skrub_impl` guaranteed to not hold a reference to a large datastructure? Calling `cloudpickle.dumps` would be problematic otherwise (by allocating a large intermediate datastructure before serializing to disk/network).

Reply:
right, that's true. no it isn't and in fact it is quite likely to hold references to dataframes that users passed when initializing a variable


all in all I think we might better off removing the pickle-ability checks, and any attempt to force usage of cloudpickle, and just recommend in the documentation that users rely on cloudpickle if pickle cannot handle their pipeline (eg due to lambda expressions)